resources/views/adm/mship/feedback/settings.blade.php

```diff
@@ -0,0 +1,207 @@
+@extends('adm.layout')
+
+@section('styles')
+{!! HTML::style(asset("assets\js\plugins\datetimepickerxdan\jquery.datetimepicker.min.css")) !!}
+<style type="text/css">
+    body.dragging, body.dragging * {
+      cursor: move !important;
+    }
+
+    .dragged {
+      position: absolute;
+      opacity: 0.5;
+      z-index: 2000;
+    }
+
+    ol.example li.placeholder {
+      position: relative;
+      /** More li styles **/
+    }
+    ol.example li.placeholder:before {
+      position: absolute;
+      /** Define arrowhead **/
+    }
+
+    ol.simple_connected_list{
+      display: inline;
+    }
+
+    ol.simple_connected_list li {
+      list-style: none;
+    }
+</style>
+@endsection
```

Comment:
At the moment this is not possible due to the Admin interface not referencing app-all.css because of its completely different interface. I have moved it into the AdminLTE styling, however.